packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -71,18 +72,37 @@ function SelectionTool(props: Props) {
       }
 
       const { worldPt, sourceEvent } = evt;
-      const newEndPoint = worldToData(boundPointToFOV(worldPt, camera));
-      setEndPoint(newEndPoint);
+      const dataEndPoint = worldToData(boundPointToFOV(worldPt, camera));
+      setEndPoint(dataEndPoint);
+
+      const worldEndPoint = dataToWorld(dataEndPoint);
+      const htmlEndPoint = cameraToHtml(
+        new Vector3(worldEndPoint.x, worldEndPoint.y, 0).project(camera)
```

Comment:
A small `worldToCamera` utility could be nice, maybe? You could use it in `dataToHtml` as well.

Reply:
I went a step further by adding `worldToHtml` to the `AxisSystemContext`

Reply:
Hmm, isn't that what we had done at some point and decided to move it out? Now, this sort of "camera-based" functions are in `src/vis/utils.ts` and take the `camera` as first parameter.

- We have `dataToWorld` and `cameraToHtml` in the context, because they don't actually use `camera`.
- The only missing piece is `worldToCamera`, which does use `camera` so should go in `src/vis/utils.ts`.
- We also have a higher-order utility `dataToHtml` also in `src/vis/utils.ts`.
- Now I don't have any objection adding `worldToHtml` in there as well if you think it makes sense.